scripts/file.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Demo script for test the funktion "file.getJSON" and "file.getJSONEncrypted".
+ *
+ * Doc for "file.getJSON" and "file.getJSONEncrypted" usage: node scripts/file.js userId pathSeed myskyJsonPath
+ *
+ * Example: node scripts/file.js "4dfb9ce035e4e44711c1bb0a0901ce3adc2a928b122ee7b45df6ac47548646b0" "fe2c5148646532a442dd117efab3ff2a190336da506e363f80fb949513dab811" "test.hns/encrypted"
+ *
+ * Example with default data: node scripts/file.js
+ *
+ */
+
+(async () => {
+  // Fill in default variables here:
+  const defaultUserId = "4dfb9ce035e4e44711c1bb0a0901ce3adc2a928b122ee7b45df6ac47548646b0";
+  const defaultPathSeed = "fe2c5148646532a442dd117efab3ff2a190336da506e363f80fb949513dab811";
+  const defaultMyskyJsonPath = "test.hns/encrypted";
+  // end default variables
+
+  const fs = require("fs");
+  var dir = "./tmp/download/";
+
+  if (!fs.existsSync(dir)) {
+    fs.mkdirSync(dir, { recursive: true });
+  }
```

Comment:
You can use the [tmp](https://www.npmjs.com/package/tmp) dependency here to simplify things, it's already in `package.json`.

Reply:
hello m-cat,

You might need help here.

I know what you mean, but I don't know how to implement it.
I'm trying but not sure.

Reply:
You can see an example in `download.test.js`. Let me know if you have any questions. 🙂